tests/app/questionnaire/routing/test_operators.py

```diff
@@ -331,3 +331,16 @@ def test_nonetype_operands_for_comparison_operators(operator_name, operands):
 def test_nonetype_operands_for_array_operators(operator_name, operands):
     operator = Operator(operator_name)
     assert operator.evaluate(operands) is False
+
+
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    [
+        ([("tuple element 1", "tuple element 2")], 2),
+        ([["list element 1", "list element 2"]], 2),
+        ([None], 0),
```

Comment:
I think it is worth adding an empty collection also.
```suggestion
        ([("tuple element 1", "tuple element 2")], 2),
        ([["list element 1", "list element 2"]], 2),
        ([[]], 0),
        ([None], 0),```